CMakeLists.txt

```diff
@@ -34,8 +34,15 @@ if(UNIX)
 endif()
 
 # set explicitly the c++ standard to use
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++03")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++03")
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O2 -march=native")
```

Comment:
Did you find problems when compiling with `-O3`? I think that O3 gives much better performance in most cases I've tried.

Reply:
Whoops. Only meant to have that on the debugging stuff.